nix/slothy/default.nix

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: Apache-2.0
+
+{ stdenvNoCC
+, fetchFromGitHub
+, python311
+, pkgs
+}:
+
+let
+  pythonEnv = python311.withPackages (ps: with ps; [
+    ortools
```

Comment:
(How) are the versions fixed here?

Reply:
currently it just takes the default version in the current version. For ortools that happens to be 9.7. 
I don't quite know how to change that - we will likely need to take it from a different channel, but that can quickly get impossible due to Python versions. In that case we would have to build it ourselves by overwriting the package definitions just like we do for cbmc.

Reply:
it's not pip, so you cannot expect to just be able to throw in a any version that is cached in pypi.

Reply:
Actually, there seems to be a `fetchPypi`, so we can probably make it work for any version in pypi somehow.